lib/History/History.js

```diff
@@ -100,6 +74,14 @@ var History = module.exports = kind.singleton({
 		this._popStateHandler = this.bindSafely(this.popStateHandler);
 
 		this.enableBackHistoryAPIChanged();
+
+		this._appInfoReq = new Ajax({url:"appinfo.json"});
+		this._appInfoReq.response(function(inSender, inResponse) {
+			if(inResponse !== undefined) {
```

Comment:
Minor code style items - add a space between `function` and `if` and the opening paren. No need to prefix arguments with `in`.